ha_reolink_rich_notifications.yaml

```diff
@@ -36,13 +36,31 @@ blueprint:
       selector:
         duration:
 
+    notification_title:
+      name: Notification Heading
+      description: Title of the notification
+      default: "{{ trigger.from_state.attributes.friendly_name }}"
+
+    notification_message:
+      name: Notification Subtext
+      description: Additional information
```

Comment:
Can you expand the description here too and say that this text will show up in the notification message body?